### ISSUE TITLE
helm.valuesFiles not being overwritten 

### ISSUE THREAD

**Issue by sorushsaghari** [rancher/fleet#405]
this is my `fleet.yaml` file 
defaultNamespace: fleet-local
```
namespace: fleet-local

helm:
  chart: https://chart-repo/helm/chart-name-0.1.3.tgz
  releaseName: my-release
 
  valuesFiles:
    - values.yaml
  
  force: false

paused: false
```
the problem is that i can use my helm chart but values are not overwritten . `values.yaml` file next to `fleet.yaml` file

**Comment by oxr463** [comment]
@sorushsaghari can you share the helm chart and `values.yml` file you are testing with?

**Comment by oxr463** [comment]
Possibly related to https://github.com/rancher/fleet/issues/384

**Comment by ricanol** [comment]
Hi, the same problem happens to me!

A little different from @sorushsaghari  problem I'm trying to install airflow via helm using fleet.yaml

(https://github.com/airflow-helm/charts/tree/main/charts/airflow)

using values.yaml from the same Airflow repository!

I can go up, but it doesn't overwrite the values.yaml I have along with my fleet.yaml

**Comment by kkaempf** [comment]
should be fixed by #516
